src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -496,11 +498,17 @@ class GridEditable<
           {/* TODO(leedongwei): This is ugly but I need to move it to work on
           resizing columns. It will be refactored in a upcoming PR */}
           <div style={{display: 'flex', flexDirection: 'row'}}>
-            {this.renderHeaderButton()}
-
-            <div style={{marginLeft: '16px'}}>
+            <HeaderButtonContainer>
```

Comment:
@wmak it might be reasonable to hide this button if there is no data. For example, the empty state.

Reply:
I did this with `isLoading`, is that what you meant with empty state? Or something else?

Reply:
Ah. I had meant when `tableData` was empty.

But, I think this is good enough.

I thought there may be issues with generating the csv data from an empty `tableData`; but it seems to import fine on Google Sheets. 👍